volontulo/tests/views/test_organizations.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+
+u"""
+.. module:: test_organizations
+"""
+from django.test import Client
+from django.test import TestCase
+
+from volontulo.models import Organization
+from volontulo.tests.views.test_usersprofile import TestUsersProfile
```

Comment:
That not a good idea to import methods from another testcase. If we must do something like this we should move common code to common.py or something like this.